wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
 
 ##### Dendens
-These appear as spinners, with a countdown displaying the time you have remaining to clear it.
--   You must hit the inner and outer drum in order (like red,red, blue,blue, red,red, and so on) until the Denden meter is full.
--   Bonus points are given for extra hits after completing the Denden.
+![Taiko spinner][Taiko spinner image]
 
-#### Controls
-The default controls for the Taiko mod are:
+These appear as spinners, with a countdown displaying the number of hits remaining (denden counter) to clear it, and a shirking blue circle showing the time left to clear it.
 
-```
-Red: {Mouse} Left click, {Keyboard} [X] or [C], {TaTaCon} Flat surface of the drum.
-Blue: {Mouse} Right click, {Keyboard} [Z] or [V], {TaTaCon} Outer surface of the drum.
+Hit the inner and outer drum _in order_ (like red,blue, red,blue, red,blue, ...) until the denden counter reached 0. Starting colour does not matter (allowed to start with blue if desired), and failing to complete it will incur miss health penalty but no combo break. Hitting the same colour will not decrement the denden counter, until a different colour was hitted instead.
 
-osu!cursor placement does not matter when playing.
-```
+It does not increment the combo counter, and does not recover the healthbar in any way. Each successful denden hit only gives a static 300 score, and denden completion will give a perfect(GREAT) large note score.
 
-You may configure the keyboard buttons as you see fit, from the Input tab in the Options menu.
+#### Controls
+The default controls for the osu!taiko are:-
 
-### What is "osu!catch"?
+Type | Mouse | Keyboard | TaTaCon
+---|---|---|---
+Red |Left click|`X` / `C`|Flat surface of the drum
+Blue|Right click|`Z` / `V`|Outer surface of the drum
+
+The placement of in-game cursor does not matter when playing.
+
+### What is osu!catch?
 _Full explanation: [osu!catch][osu!catch wikilink]_
 
 ![osu!catch Interface][osu!catch Interface image]
 
-_osu!catch_ (previously known as _Catch the Beat_) is the second special game mode to be added in, after osu!taiko game mode.
+_osu!catch_ (previously known as _Catch the Beat_) is the second game mode added to osu!, after osu!taiko game mode.
 
 Instead of clicking circles, you are now tasked to move the catcher at the bottom of the screen while the fruits fall from above.
 
+#### Basic explanation
+All fruits will stay at the plate until its combo ends, which the fruits will bounce out of the plate. Timing and where it falls on the plate does not matter; either a full score for successful catch or miss.  
+
+##### Fruits
+![Fruits][Fruits image]
+
+Fruits are the hitcircles of osu!catch, and it rains down from top of the screen. To catch these fruits, make sure it falls to the plate, and not over the plate. Collected fruits will always give 300pt each, increment combo by 1, and treated as 300 in result screen.
```

Comment:
use `points` instead of `pt`, if you want to keep the abbreviation, then use `pts` ... I guess?